README-code-reviews.md

```diff
@@ -0,0 +1,103 @@
+* If the PR source branch hasn't synced up from master in a long time, or if there is an upstream commit not in the source branch that might interfere with the PR, ask the PR author to merge the latest upstream master.
+* Test the code locally if you have any concerns as to whether and how it works; as a helpful tip, merge the latest upstream master into your locally fetch PR review branch before testing
+
+#### Before you integrate or sponsor a PR:
+
+Skara will mark a PR as "ready" once the minimum number of reviewers have reviewed it. Before you integrate or sponsor the PR, ensure that the following have been done:
+
+* All substantive feedback has been addressed, especially any objections from one with a Reviewer role.
+* All Reviewers who have requested the chance to review have done so (or indicated that they are OK with it going in without their review). In rare cases a Project Lead may override this.
+* The PR has been "rfr" (as indicated by Skara) for at least 1 business day (24 hours), not including weekends / or major holidays. This is to allow sufficient time for those reviewers who might be in other time zones the chance to review if they have concerns. This is measured from the time that Skara has most recently added the "rfr" label (for example, for a PR that was previously in Draft mode, wait for at least 24 hours after the PR has been taken out of Draft and marked "rfr"). In rare cases (e.g., a build breakage) a Reviewer might give the OK to integrate without waiting for 24 hours.
+* Double-check the commit message before you integrate. Skara will list it near the top of your PR.
+
+#### A. Low-impact bug fixes.
+
+These are typically isolated bug fixes with little or no impact beyond fixing the bug in question; included in this category are test fixes (including most new tests), doc fixes, and fixes to sample applications (including most new samples).
+
+One (1) "R"eviewer is usually sufficient to accept such changes.
+
+#### B. Higher impact bug fixes or RFEs.
+
+These include changes to the implementation that potentially have a performance or behavioral impact, or are otherwise broad in scope. Some larger bug fixes will fall into this category, as will any fixes in high-risk areas (e.g., CSS).
+
+At least two (2) reviewers must approve to accept such changes, at least one of whom must be a "R"eviewer.
+
+#### C. New features / API additions.
+
+This includes behavioral changes, additions to the FXML or CSS spec, etc.
+
+Feature requests come with a responsibility beyond just saying "here is the code for this cool new feature, please take it". There are many factors to consider for even small features. Larger features will need a significant contribution in terms of API design, coding, testing, maintainability, etc.
+
+A feature should be discussed up-front on the openjfx-dev mailing list to get early feedback on the concept (is it a feature we are likely to accept) and the direction the API and/or implementation should take.
```

Comment:
are there any special rules to gauging the consensus?  what happens when one party in a discussion stops responding?